src/workerd/api/sockets.h

```diff
@@ -17,9 +17,9 @@ struct SocketAddress {
 };
 
 struct SocketOptions {
-  jsg::Unimplemented tls; // TODO(later): TCP socket options need to be implemented.
+  bool useSecureTransport;
```

Comment:
```suggestion
  bool useSecureTransport = false;
```

Or wrap it in a `jsg::Optional`